README.md

```diff
@@ -1,2 +1,14 @@
-# DPS909_Open_Source
-This repo is for Open source project in BSD.
+# URL-Inspector
+This is a command line interface tool to check different URLs and detect if they are good or bad or unknow.
+
+# Instalation
+npm i g https://github.com/chawlapalak/Url-Inspector
+
+# Features
+
+- You can check the different URLs by runing this script if they are working or not : url-inspector -f test.txt.
+- It will display good in green , if it is working.
```

Comment:
It's more understandable if the status codes are shown. For eg, 400/404 - bad, 200 - good, other- unknown, etc.